common/src/main/java/org/apache/sedona/common/raster/netcdf/SedonaNetCdfReader.java

```diff
@@ -0,0 +1,359 @@
+package org.apache.sedona.common.raster.netcdf;
+
+import com.google.common.collect.ImmutableList;
```

Comment:
Where does this `com.google.common` come from? The imports here are really messy.

Reply:
This is being used internally by the unidata Java NetCDF packages. I am using this to be able to get a ImmutableList (which is also the original datatype being used in the netCDF package).

I understand that using ImmutablePair of apache common is unnecessary, I've instead returned an array of Variables and removed apache commons import.